class.jetpack.php

```diff
@@ -669,6 +670,15 @@ function point_edit_links_to_calypso( $default_url, $post_id ) {
 		return esc_url( sprintf( 'https://wordpress.com/%s/%s/%d', $path_prefix, $site_slug, $post_id ) );
 	}
 
+	function point_edit_comment_links_to_calypso( $link, $comment_id, $text ) {
+		$url = wp_parse_url( get_home_url() );
+		$html = sprintf( '<a class="comment-edit-link" href="%s">%s</a>',
```

Comment:
Once we switch to [`get_edit_comment_link`](https://developer.wordpress.org/reference/hooks/get_edit_comment_link/) filter, we can simplify this part - we'll only need to return the `href`.

Reply:
Thanks @vindl ! Updated in 64643f843295489aae584e245d2c3c9341314f45.